src/reports/components/instance-details.scss

```diff
@@ -14,12 +14,39 @@
 
 .instance-details-card {
     border-radius: 4px;
+    outline-style: 'border-style';
     box-shadow: 0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132);
     margin-left: 24px;
     margin-bottom: 16px;
     @include fill-available-width;
 }
 
+.instance-details-card.selected {
+    border: 5px solid #2b88d8;
+}
+
+.instance-details-card:focus {
+    outline: 2px solid rgba(0, 0, 0, 0.9);
+    cursor: pointer;
```

Comment:
`cursor: pointer;` is repeated across several "sub-styles" of `.instance-details-card`, can it be applied universaly, thus avoiding the duplication?